tests/wasm/classes.rs

```diff
@@ -480,6 +480,19 @@ fn renamed_field() {
     js_renamed_field();
 }
 
+#[cfg_attr(
+    target_arch = "wasm32",
+    wasm_bindgen(inspectable, js_name = "ConditionalSkipClass")
```

Comment:
```suggestion
    wasm_bindgen(js_name = "ConditionalSkipClass")
```
Or is there some reason this is here?